cloudigrade/account/models.py

```diff
@@ -0,0 +1,15 @@
+"""Cloudigrade Account Models."""
+from django.db import models
+
+from util.models import BaseModel
+
+
+class Account(BaseModel):
+    """Account model."""
+
+    account_id = models.DecimalField(
```

Comment:
When you have 2 columns, this seems trivial, but I've seen a directory of models grow to 100+ models. It can be really nice to have a short comment on each column giving some specifics. Say if there's a check constraint, it's a good place to put the possible/expected values and what they mean. 

Here I would consider just a quick note that these are currently AWS specific.

Reply:
I think AWS is kinda assumed at this point. account_arn wouldn't make much sense for gcp or azure, when we add support for other clouds I see this table having some drastic changes, but I don't think it makes sense currently to add any aws specific distinctions yet.

Reply:
Sure, I stand by my usefulness of column comments though. As you add more people looking at the code base it's just a service to newcomers.